### PR TITLE
fixes #194 add underground tag to buildings (and addr points)

### LINKED ISSUE #194: add underground tag to buildings
There are a couple of BART stations which are underground: https://www.openstreetmap.org/way/155378773
https://www.openstreetmap.org/way/160331904

But the tiles have them as just vanilla buildings at street level: 
http://tangrams.github.io/explorer/#17.5/37.7896/-122.4013/name/Montgomery%20Stat

https://trello-attachments.s3.amazonaws.com/55f1d2933cfa613ec481fe6c/978x816/e9e8d46c82c10fdd49e3d4af117272b5/Screen_Shot_2015-09-10_at_2.55.37_PM.png

### PR DESCRIPTION
**Pull request by nvkelso**
Adds `layer` and `location` to building features (from tags as those aren't included in the default import style file), and since the buildings layer includes address points also adds `addr_floor` and `level` (from tags).

### CODE REVIEW

File: queries/buildings.jinja2 [hunk 1]
```diff
@@ -73,6 +79,8 @@ SELECT
     NULL AS roof_orientation,
     "addr:housenumber" AS addr_housenumber,
     "addr:street" AS addr_street,
+    tags->'addr:floor' AS addr_floor,
+    tags->'level' as level,
```
Comment (zerebubuth) [inline queries/buildings.jinja2]: Taginfo suggests that only [0.02% of nodes](http://taginfo.openstreetmap.org/keys/addr%3Ahousenumber?filter=nodes#combinations) tagged with `addr:housenumber` also have `level` set. Were there some specific nodes you were looking at where `level` was important?

Comment (nvkelso) [inline queries/buildings.jinja2]: Usually its not a problem until it is-large apartment towers were suddenly there are too many address points to display.

If this introduces too much cost to the query we should drop it for now, was trying to edge case / future proof.

_n

Comment (zerebubuth) [inline queries/buildings.jinja2]: Sorry, I didn't make my point very well. What I meant to say was: this seems like a very rarely-used tag, that doesn't provide a lot of extra information in the vast majority of cases. If you've got some examples of large apartment blocks with too many address points then we should check to see if `level` has been defined on any of those, so we can check that it'll provide the information for the edge-cases that you want it for.

Comment (nvkelso) [inline queries/buildings.jinja2]: Let's drop the add_floor and level bits from this until there's more of a use case.

**Comment by zerebubuth** [comment]
:+1: looks good to me.

**Pull request merged**